plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1248,7 +1247,7 @@ func PodToleratesNodeTaints(pod *v1.Pod, meta interface{}, nodeInfo *schedulerca
 	return false, []algorithm.PredicateFailureReason{ErrTaintsTolerationsNotMatch}, nil
 }
 
-// Determine if a pod is scheduled with best-effort QoS
+// isPodBestEffort checks if pod is scheduled with best-effort QoS
```

Comment:
Please note there is a PR covering the godoc for all of the public api here https://github.com/kubernetes/kubernetes/pull/47052